docs/tasks/timeout-injection.md

```diff
@@ -0,0 +1,116 @@
+# Injecting Timeouts
```

Comment:
Still need to use the templates. If the {% capture %} {% endcapture %} annotations you can see in the other documents in istio.github.io.